deploy/osd-cluster-ready/job/10-osd-ready.Job.yaml

```diff
@@ -13,13 +13,13 @@ spec:
             name: osd-cluster-ready
             labels:
                 # Keep this in sync with image
-                managed.openshift.io/version: "v0.1.64-f55bb06"
+                managed.openshift.io/version: "v0.1.66-bb79af5"
         spec:
             containers:
             - name: osd-cluster-ready
               # Keep the `managed.openshift.io/version` label in sync
               # with this
-              image: "quay.io/openshift-sre/osd-cluster-ready@sha256:1e977ed7e1ceab93b64a8da51ed41fdc9ccfb06b1ce50b29b4d77f9eb622b680"
+              image: "quay.io/openshift-sre/osd-cluster-ready@sha256:fa4aa5826ad0a8740604a8f50034d8ff80263cbf5eb339e416ee2bfa23c09dc6"
```

Comment:
Over on quay it looks like the sha256 sum is `quay.io/openshift-sre/osd-cluster-ready@sha256:84d0c91e5df7455f164eb6cc5544a09e271e94e5c7d0a7644332de69127c8437`
can we double check this?